src/ui/CategoryNav/CategoryNav.jsx

```diff
@@ -0,0 +1,13 @@
+import "./CategoryNav.css";
+import { CategoryNavButtons } from "./CategoryNavButtons/CategoryNavButtons";
+
+export function CategoryNav({ activeCategoryName }) {
```

Comment:
P3:
카테고리 명을 받아 "전체"가 아닐때 옵션 버튼들을 보여주는 것보다
외부에서 필요에 따라 옵션이 보이는 조건이 변경 될 수 있도록 하면 좋겠네요.

```suggestion
export function CategoryNav({ categoryName, isShowOptions }) {
```